src/SFA.DAS.EmployerFinance/Configuration/GoogleAnalyticsConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+namespace SFA.DAS.EmployerFinance.Configuration
+{
+    public class GoogleAnalyticsConfiguration
```

Comment:
This will need registering in the container.

Reply:
done